data/human/ships.txt

```diff
@@ -1621,7 +1621,7 @@ ship "Falcon"
 	sprite "ship/falcon"
 	thumbnail "thumbnail/falcon"
 	attributes
-		category "Heavy Warship"
+		category "Light Warship"
```

Comment:
A Falcon has got a lot of durability for LW.

Reply:
It's relatively fragile compared to similarly sized ships. The decisions made in its design seems to be deliberately leaning towards speed at the expense of toughness, which makes it a 'light warship' in the terms in this PR.